README.md

```diff
@@ -54,7 +56,7 @@ Now, your Planet API Key is stored in the variable ```PLANET_API_KEY``` and is r
 ## Sentinel Hub Python SDK
```

Comment:
With the changes to SDK notebook auth, it might be worth adjusting the API key section above, since that will not always need the API key anymore - or at least noting that there's now two supported Planet Auth mechanisms.